HTML+CSS+JS/README.md

```diff
@@ -2,6 +2,10 @@
 
 ## Table of contents -
 
+-   [Animation](Animation)
+
+    -   [SVG Bounce Animation Component](Animation)
```

Comment:
`[SVG Bounce Animation Component](Animation)` link should be `[SVG Bounce Animation Component](Animation/SVG Bounce animation)`

I'd suggest rename this folder and remove space in this folder name `SVG Bounce animation` 
- write the folder name in Camel case or snake case, space in folder name is usually avoided.
- Since no SVG used in the component hence we can rename it to `ImageBounceAnimation`